### ISSUE TITLE
Don't work with ipega PG-9156

### ISSUE THREAD

**Issue by MichelCibrario** [imurvai/brickcontroller2#12]
The Controller tester don't detect the controller ipega PG-9156 with:
SAMSUNG NOTE 9
SAMSUNG NOTE 3

**Comment by imurvai** [comment]
Hi, did you pair the controller in the Android's bluetooth settings?

**Comment by MichelCibrario** [comment]
Yes. It's paired to the phone with the bluetooth menu. The ipega works with the phone. Ps4 controller work fine.

**Comment by imurvai** [comment]
Most bluetooth gamepads have a couple of operating modes which can be selected by pressing a button on it. It is depending on the model, so please check the ipega's manual how to change the mode. I have a cheap bt gamepad and it has 4 modes and only one is working as a gamepad.

**Comment by MichelCibrario** [comment]
IT work!!!!!! Mode home+X (standard mode) thanks !!!!
 
Cordialy
 
 
 
 
> Message du 02/09/19 20:10> De : "István Murvai" > A : "imurvai/brickcontroller2" 
> Copie à : "MichelCibrario" , "Author" > Objet : Re: [imurvai/brickcontroller2] Don't work with ipega PG-9156 (#12)> >
> Most bluetooth gamepads have a couple of operating modes which can be selected by pressing a button on it. It is depending on the model, so please check the ipega's manual how to change the mode. I have a cheap bt gamepad and it has 4 modes and only one is working as a gamepad.
—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.
[ { "@context": "http://schema.org", "@type": "EmailMessage", "potentialAction": { "@type": "ViewAction", "target": "https://github.com/imurvai/brickcontroller2/issues/12?email_source=notifications\u0026email_token=ANCATLLQB2PSOMTQXUTF2Y3QHVJK5A5CNFSM4ISZUZP2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD5WLFCA#issuecomment-527217288", "url": "https://github.com/imurvai/brickcontroller2/issues/12?email_source=notifications\u0026email_token=ANCATLLQB2PSOMTQXUTF2Y3QHVJK5A5CNFSM4ISZUZP2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD5WLFCA#issuecomment-527217288", "name": "View Issue" }, "description": "View this Issue on GitHub", "publisher": { "@type": "Organization", "name": "GitHub", "url": "https://github.com" } } ]
